Build broken on FreeBSD

```
error[E0425]: cannot find value `CLOCK_BOOTTIME` in crate `libc`
   --> /root/.cargo/registry/src/github.com-1ecc6299db9ec823/sysinfo-0.14.1/src/linux/system.rs:121:43
    |
121 |     if unsafe { libc::clock_gettime(libc::CLOCK_BOOTTIME, &mut up) } == 0 {
    |                                           ^^^^^^^^^^^^^^ help: a constant with a similar name exists: `CLOCK_REALTIME`
```
It looks like you added a dependency that doesn't support it. If FreeBSD is being dropped forever then perhaps make a note about this in the readme. This is a shame though because it was working well for me before.

--- a/jormungandr-lib/Cargo.toml
+++ b/jormungandr-lib/Cargo.toml
@@ -22,7 +22,7 @@ chrono = { version = "0.4", features = ["serde"] }
 humantime = "2.0"
 thiserror = "1.0"
 poldercast = "0.11.4"
-sysinfo = "0.14.1"
+sysinfo = { version = "0.14.1", optional = true }
 os_info = { version = "2.0.0", default-features = false }
 zip = "0.5.5"
 flate2 = "1.0.14"
@@ -38,6 +38,7 @@ chain-crypto    = { path = "../chain-deps/chain-crypto", features = [ "property-
 ed25519-bip32 = "0.3"
 serde_yaml = "0.8"
 bincode = "1.1"
+sysinfo = "0.14.1"
 
 [target.'cfg(not(target_os = "android"))'.dependencies.reqwest]
 version = "0.10.4"
@@ -54,4 +55,4 @@ features = ["blocking"]
 
 [features]
 default = []
-property-test-api = [ "chain-impl-mockchain/property-test-api", "chain-addr/property-test-api" ]
+property-test-api = [ "chain-impl-mockchain/property-test-api", "chain-addr/property-test-api", "sysinfo" ]

